Source/WebKit/ModelProcess/cocoa/ModelProcessModelPlayerProxy.mm

```diff
@@ -352,11 +350,31 @@ static CGFloat effectivePointsPerMeter(CALayer *caLayer)
     RECALayerClientComponentSetShouldSyncToRemotes(layerComponent.get(), true);
 
     auto clientComponent = RECALayerGetCALayerClientComponent(m_layer.get());
-    auto rootEntity = REComponentGetEntity(clientComponent);
-    REEntitySetName(rootEntity, "WebKit:ClientComponentRoot");
+    auto clientComponentEntity = REComponentGetEntity(clientComponent);
+    REEntitySetName(clientComponentEntity, "WebKit:ClientComponentRoot");
```

Comment:
Let's also call it "WebKit:ClientComponentEntity"

Reply:
Done.